### PR TITLE
feat: added 'Drive with Google' button to route example

### PR DESCRIPTION
**Pull request by Khrystyna**
Added "Drive with Google" button that exports route to Google Maps.

### CODE REVIEW

File: route/index.js [hunk 1]
```diff
@@ -139,3 +145,29 @@ const displayRouteData = data => {
...
 };
+
+const getGoogleMapDirectionsURL = legs => {
+  if (legs.length === 0) return;
+
+  let googleDirURL = `https://google.com/maps/dir/?api=1`;
+  const origin = legs[0].origin?.geometry?.coordinates;
+  const destination = legs[legs.length - 1].destination?.geometry?.coordinates;
+
+  // coordinates is an array with longitude as first value and latitude as second one
```
Comment (lsmienk) [inline route/index.js:156]: coordinates are an array with longitude as first value and latitude as the second one

File: route/index.js [hunk 2]
```diff
@@ -139,3 +145,29 @@ const displayRouteData = data => {
   // the total amount of CO2 which were used with a petrol vehicle
   document.getElementById('co2').innerHTML = data.saving?.co2 ? `${data.saving.co2 / 1000} Kg` : 'Unknown';
 };
+
```
Comment (lsmienk) [inline route/index.js:148]: Can we add a comment about the entire function?

File: public/route/index.html [hunk 3]
```diff
@@ -80,6 +80,8 @@ <h2>Trip information</h2>
             <span id="co2">--•--</span>
           </li>
         </ul>
+
+        <button id="drive" disabled>Drive with Google Map</button>
```
Comment (lsmienk) [inline public/route/index.html:84]: Is it not Google Maps?

**Review by lsmienk — COMMENTED** [review]
Looks good! just a few suggestions about the comments

File: route/index.js (hunk 1 above)
Comment (Khrystyna) [reply route/index.js:156]: :thumbUp:

**Review by lsmienk — APPROVED** [review]

**Pull request merged**